libs/components/core/src/lib/modules/live-announcer/live-announcer.ts

```diff
@@ -0,0 +1,172 @@
+import { DOCUMENT } from '@angular/common';
+import { Injectable, OnDestroy, inject } from '@angular/core';
+
+import { ReplaySubject } from 'rxjs';
+
+import { SkyIdService } from '../id/id.service';
+
+import {
+  SKY_LIVE_ANNOUNCER_DEFAULT_OPTIONS,
+  SKY_LIVE_ANNOUNCER_ELEMENT_TOKEN,
+  SkyAriaLivePoliteness,
+  SkyLiveAnnouncerDefaultOptions,
+} from './live-announcer-tokens';
+
+@Injectable({ providedIn: 'root' })
+export class SkyLiveAnnouncer implements OnDestroy {
+  public static announcerElement: HTMLElement | undefined;
```

Comment:
I thought that we decided during strict mode that static members were inherently public and so they should be listed that way? 

Either way - at least the ReplaySubject needs to be public as modal's use it. The other is only used by the live announcer service(s) but I listed it as public as I thought we made all static things public when we did strict mode.

Reply:
Since this is provided in root, I don't think we need any static members. If it turns out we still need it, we can declare them outside the class so they're not public. Also if we have static members that are used outside this class, then that's a code smell because you lose the benefits of dependency injection.

Reply:
I have removed the use of static members.